### PR TITLE
Selenium branch

### PR DESCRIPTION
**Pull request by OverlordDen**
Tried to do some Page class

### CODE REVIEW

File: Projects/Team1/Semeniuk/FeatureSeleniumLogin/SeleniumTests/SeleniumTests/Program.cs [hunk 1]
```diff
@@ -44,24 +43,31 @@ public void FirstTest()
...
+            page.loginPageButton.Click();
+
             //e-mail
-            driver.FindElement(By.Id("login-email")).SendKeys(email);
+            page.email.SendKeys(email);
+
             //password
-            driver.FindElement(By.Id("login-password")).SendKeys(password + Keys.Enter);
+            page.password.SendKeys(password + Keys.Enter);
             Thread.Sleep(2000);
```
Comment (romanukhaav) [inline Projects/Team1/Semeniuk/FeatureSeleniumLogin/SeleniumTests/SeleniumTests/Program.cs]: Could you remove Thread.Sleep ?

Comment (OverlordDen) [reply Projects/Team1/Semeniuk/FeatureSeleniumLogin/SeleniumTests/SeleniumTests/Program.cs]: Got it

File: Projects/Team1/Semeniuk/FeatureSeleniumLogin/SeleniumTests/SeleniumTests/Program.cs [hunk 2]
```diff
@@ -7,61 +7,72 @@
...
+            page.Login();
+            
+            //login
+            page.cofirmLogin.Click();
+
             //Checking for new(after login) functions
             driver.Navigate().GoToUrl(personalOffice);
-            Thread.Sleep(2000);
-            IWebElement actual = driver.FindElement(By.XPath("/html/body/div[4]/div/div/div[3]/center/div/a"));
+            //Thread.Sleep(2000);
```
Comment (romanukhaav) [inline Projects/Team1/Semeniuk/FeatureSeleniumLogin/SeleniumTests/SeleniumTests/Program.cs:72]: remove comments please

**Review by romanukhaav — APPROVED** [review]

**Comment by romanukhaav** [comment]
Excellent!

**Pull request merged**